internal/commands/loglevel/show.go

```diff
@@ -52,12 +52,14 @@ func showLogSettings(ctx context.Context) error {
 // Show displays information about log verbosity (if set)
 func Show(ctx context.Context, args []string) error {
 	doc := `Usage:
-  sveltosctl log-level show
+  sveltosctl log-level show --namespace=<namespace> --cluster=<cluster-name>
```

Comment:
same comments as above:

1. [--namespace=<namespace>] [--cluster=<cluster-name>] cause those are optional
2. add cluster-type
3. indentation for the comment of each option should be same